frontend/CLAUDE.md

```
@@ -382,6 +386,17 @@ const routes = {
 }
 ```
 
+#### Community POAPs
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)